include/spblas/concepts.hpp

```diff
@@ -17,9 +17,8 @@ namespace spblas {
 */
 
 template <typename M>
-concept matrix =
-    __detail::is_csr_view_v<M> || __detail::is_csc_view_v<M> ||
-    __detail::is_matrix_instantiation_of_mdspan_v<M> || __detail::matrix<M>;
+concept matrix = __detail::is_csr_view_v<M> || __detail::is_csc_view_v<M> ||
+                 __detail::is_matrix_mdspan_v<M> || __detail::matrix<M>;
```

Comment:
I am guessing the "is_xyz_view_v" look through the matrix_opt to see the xyz_view underneath so a matrix_opt would also be classified properly here ...